File: Auto-generated-tasks/Difficulty-2/Task-8/Solution.go
package "main"
import "fmt"
import "math/rand"
Num := 6
u := 0
A2 := 0
var Mas[Num] rune
for u = 0 u < Num u++ {
	Mas[u] = Intn(434) + 4
	fmt.PrintLn("Mas[", u, "]=", Mas[u])
	A2 = A2 + Mas[u]
}
fmt.PrintLn("A2=", A2)

File: Auto-generated-tasks/Difficulty-2/Task-8/Solution.f90
integer:: Num = 6, u, A2 = 0
integer(len=Num):: Mas
randomize()
do u = 0, Num
	call random_number(r)
	Mas(u) = (435 * r) + 4;
	if (Mas(u) > 438) Mas(u) = 438
	print *,"Mas[", u, "]=", Mas(u)
	A2 = A2 + Mas(u)
end do
print *, "A2=", A2